packages/calypso-products/src/constants/features.ts

```diff
@@ -180,6 +180,8 @@ export const FEATURE_JETPACK_ANTI_SPAM = PRODUCT_JETPACK_ANTI_SPAM;
 export const FEATURE_JETPACK_ANTI_SPAM_MONTHLY = PRODUCT_JETPACK_ANTI_SPAM_MONTHLY;
 export const FEATURE_JETPACK_SEARCH = PRODUCT_JETPACK_SEARCH;
 export const FEATURE_JETPACK_SEARCH_MONTHLY = PRODUCT_JETPACK_SEARCH_MONTHLY;
+export const FEATURE_JETPACK_STATS_PAID = 'jetpack-stats-paid';
```

Comment:
Should we aline features with products as [other plugins](https://github.com/Automattic/wp-calypso/blob/add/jetpack-stats-products/packages/calypso-products/src/constants/features.ts#L181-L182) do? We could iterate it when the plan support gets clearer.